src/components/input-icon-toggle/input-icon-toggle.scss

```diff
@@ -0,0 +1,62 @@
+@import "./../../style-config/colors";
```

Comment:
We should be building new components using Styled Components and not scss.

Reply:
There is no possibility to use variables and methods from scss in styled components without exporting them, we also have to keep the backward compatibility with previous implementation.

Reply:
For sure, but we'll need the styled components implementation also.

Reply:
Wondering whether its safer and more valuable if we leave Date and Dropdown using the old implementation, and rather set the new components up within the new input components under /__experimental - then you have no ties to the Sass implementation and can use the flexbox layout etc which would be more beneficial?

Reply:
There may be some cross over with the work @samhinton88 is doing to update the new input to use styled-components - maybe it would be easier if we consolidate what we have so far and the styling can be set up as part of his PR? Can discuss this morning after standup

Reply:
we will also need some documentation regarding usage of the style themes